src/main/java/hudson/plugins/git/browser/GitRepositoryBrowser.java

```diff
@@ -19,10 +19,16 @@
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.net.UnknownHostException;
+import java.util.Collections;
+import java.util.Objects;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.regex.Pattern;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Other places in the plugin use `org.apache.commons.lang` rather than `org.apache.commons.lang3`.  Is there a reason for the different here?
```suggestion
import org.apache.commons.lang.StringUtils;
```

Reply:
No, actually I banned that import in my IntelliJ setup (and in my plugins I have an architecture test that checks that the old library is not used anymore). But the functionality on that level will be the same.